File: ympa_fsharp_functions_server/src/model/GetAllOffersResponse.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.OfferDTO

module GetAllOffersResponse =

  //#region GetAllOffersResponse

  [<CLIMutable>]
  type GetAllOffersResponse = {
    [<JsonProperty(PropertyName = "offers")>]
    Offers : OfferDTO[];
  }

  //#endregion

File: src/gatling/scala/org/openapitools/client/model/GetAllOffersResponse.scala

package org.openapitools.client.model


case class GetAllOffersResponse (
    /* Список предложений магазина. */
    _offers: List[OfferDTO]
)
object GetAllOffersResponse {
    def toStringBody(var_offers: Object) =
        s"""
        | {
        | "offers":$var_offers
        | }
        """.stripMargin
}
